prov/tcp/src/tcpx_init.c

```diff
@@ -98,6 +108,22 @@ static void tcpx_init_env(void)
 		port_range.low  = 0;
 		port_range.high = 0;
 	}
+
+	fi_param_get_int(&tcpx_prov, "tx_size", &tx_size);
+	if (tx_size < 0) {
+		FI_WARN(&tcpx_prov, FI_LOG_EP_CTRL,"User provided "
+			"tx context size invalid. Ignoring. \n");
+	} else {
+		tcpx_default_tx_size = tx_size;
+	}
+
+	fi_param_get_int(&tcpx_prov, "rx_size", &rx_size);
+	if (rx_size < 0) {
```

Comment:
rx_size and tx_size are uninitialized in this function.  If the parameter is not defined, they will be used uninitialized.  I believe this is the reason for the CI failures.

Reply:
That's a good catch. Thanks for the careful review. 
I misinterpreted the behavior of `fi_param_get_xxx()`.  The new patch revision fixes the bug and simplifies the code.